examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    workflow =  nvt.Workflow(["item_id"] + (['item_id', 'item_brand', 'item_category', 'item_shop'] &gt;&gt; TransformWorkflow(nvt_wkflow.get_subworkflow("item")) &gt;&gt; PredictTensorflow(model_tt.first.item_block())))
</pre><p>This step is not really intuitive from user perspective. we should explain why we have two times <code>item_id</code>?  why we need to do <code>fit_transform</code> not only <code>transform</code>? why are we fitting again?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
This line and comment have us way down a rabbit-hole fixing underlying issues that make this awkward 😅